design-proposals/klip-55-tablespaces.md

```diff
@@ -1,3 +1,210 @@
 # KLIP 55 - Tablespaces
```

Comment:
disappointed about the name :)   All joking aside, I think it's a good opportunity to emphasize the "same but different" nature of ksqlDB by making reference to a familiar construct while also suggesting it may be a little different than what you're used to

Reply:
Yeah I feel the same. An ever-so-slight riff on the norm.

Reply:
`STREAMSPACE` ftw